tests/models/torchvision_models/mask_rcnn/test_predict.py

```diff
@@ -7,7 +7,9 @@
 @pytest.fixture
 def sample_dataset(samples_source):
     images_dir = samples_source / "images"
-    images_files = get_image_files(images_dir)[-2:]
+    images_files = [
```

Comment:
perhaps we could use simpler `get_files(images_dir, extensions=".jpg")[:2]`?

Reply:
Done, thanks for the suggestion!